resources/views/messages/welcome.blade.php

```diff
@@ -1 +1 @@
-<?php echo $email->message->body ?>
+<?php echo $body; ?>
```

Comment:
no i just haven't gotten around to dealing with these templates in a real way. just wanted to get the system in place...theses templates will need some of their own work cause we will probably be injecting more stuff to them and will wanna lay it out.